src/ifk_scb_compilations/objects/passenger_transport.py

```diff
@@ -0,0 +1,127 @@
+"""Inputs for request, and analysis."""
+
+from dataclasses import dataclass
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+@dataclass
+class Request_input:
```

Comment:
I'm used to the CapWords convention 

```suggestion
class RequestInput:
```

for python classes [pep 8](https://peps.python.org/pep-0008/#class-names).